examples/plugins/main.go

```diff
@@ -0,0 +1,16 @@
+package main
+
+import (
+	"fmt"
+	"plugin"
+)
+
+func main() {
+	fmt.Printf("main started...\n")
+	p, _ := plugin.Open("plugin.so")
```

Comment:
@davecheney updated, added error checks back in and trimmed how much code is included in the slide (so that it fits)